cleanlab/multiannotator.py

```diff
@@ -275,26 +313,425 @@ def get_label_quality_multiannotator(
                 annotator_agreement,
             )
 
-    if return_detailed_quality and return_annotator_stats:
-        return {
-            "label_quality": label_quality,
-            "detailed_label_quality": detailed_label_quality,
-            "annotator_stats": annotator_stats,
-        }
-    elif return_detailed_quality:
-        return {
-            "label_quality": label_quality,
-            "detailed_label_quality": detailed_label_quality,
-        }
-    elif return_annotator_stats:
-        return {
-            "label_quality": label_quality,
-            "annotator_stats": annotator_stats,
-        }
+    labels_info = {
+        "label_quality": label_quality,
+    }
+
+    if return_detailed_quality:
+        labels_info["detailed_label_quality"] = detailed_label_quality
+    if return_annotator_stats:
+        labels_info["annotator_stats"] = annotator_stats
+    if return_weights:
+        labels_info["model_weight"] = model_weight
+        labels_info["annotator_weight"] = annotator_weight
+
+    return labels_info
+
+
+def get_label_quality_multiannotator_ensemble(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray],
+    pred_probs: np.ndarray,
+    *,
+    calibrate_probs: bool = False,
+    return_detailed_quality: bool = True,
+    return_annotator_stats: bool = True,
+    return_weights: bool = False,
+    verbose: bool = True,
+    label_quality_score_kwargs: dict = {},
+) -> Dict[str, Any]:
+    """Returns label quality scores for each example and for each annotator.
```

Comment:
```suggestion
    """Returns label quality scores for each example and for each annotator, based on predictions from an ensemble of models.
```

Reply:
I don't think we ever want to have two different functions whose one-liner description is exactly the same.